python-logic/agent.py

```diff
@@ -0,0 +1,20 @@
+from rl.agents import DQNAgent
+from rl.policy import BoltzmannQPolicy
+from rl.memory import SequentialMemory
+
+class Agent:
+
+    def __init__(self, model):
+        dqn = self.build_agent(model.model, model.actions)
+        dqn.compile(Adam(lr=1e-3), metrics=['mae'])
+        dqn.fit(env, nb_steps=50000, visualize=False, verbose=1)
```

Comment:
i think the fit has to be in separate funtion and not in the constructor
(if we want to fit the same agent twice for example)